study.md

```diff
@@ -25,7 +25,10 @@ In your own words, define what a SPA is. In your answer, be sure to cite any
 relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+A SPA is a website with a single page, split out into multiple views to represent
+"pages" or "screens". It uses json/ajax to update what is displayedo n screen
+rather than refreshing
+or actually navigating to new pages
```

Comment:
Careful here, it uses AJAX typically only once to get the data from the server, then it dynamically updates the HTML without making more requests to the back end.